internal/operands/vm-console-proxy/reconcile.go

```diff
@@ -228,30 +230,28 @@ func reconcileDeploymentFuncs(deployment apps.Deployment) common.ReconcileFunc {
 }
 
 func isEnabled(request *common.Request) bool {
-	if request.Instance.GetAnnotations() == nil {
-		return false
-	}
-	enabledStr, ok := request.Instance.GetAnnotations()[EnableAnnotation]
-	if !ok {
-		return false
-	}
-	isEnabled, err := strconv.ParseBool(enabledStr)
-	if err != nil {
-		return false
+	if request.Instance.GetAnnotations() != nil {
+		enable, isFound := request.Instance.GetAnnotations()[EnableAnnotation]
+		if isFound {
+			isEnabled, err := strconv.ParseBool(enable)
+			if err == nil {
+				return isEnabled
+			}
+		}
```

Comment:
I think this change is not needed. Early `return`s are easier to read than nested `if`s.

Reply:
I agree, but isn't makes us to add another unit test, for that early return?

Reply:
In general, unit tests should check the **functionality** of a code "unit", usually a `struct` and it's public methods.
So a code change that does not change functionality, should not change unit tests. In fact unit tests are useful to verify that the changed code has the same functionality.